### PR TITLE
✨ Create Wallet Table Component

### PR DESCRIPTION
**Pull request by mrana17**
- will be used on Wallet page
- add css
- placeholder data
- install react-table

### CODE REVIEW

File: components/table/table.module.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.table {
+  background-color: var(--table-color);
+  color: var(--primary-font-color);
+  box-shadow: 0 0.2rem 0.4rem 0 #00000029;
```
Comment (fre-ben) [inline components/table/table.module.css]: Just a quick idea - if you are going to use this box-shadow more often, you could also store it in a global variable

File: components/table/table.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import styles from "./table.module.css";
+
+export type TableProps = {
+  idTitle: string;
+  addressTitle: string;
+  saleTitle: string;
+  id: string;
```
Comment (fre-ben) [inline components/table/table.tsx]: Shouldn't this be a `number`?

File: components/table/table.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  address: string;
+  sale: string;
```
Comment (fre-ben) [inline components/table/table.tsx]: Same as above - wouldn't a `number` type be better?
For calculations etc.?

File: server/samples.http [hunk 4]
```diff
@@ -3,7 +3,7 @@ POST http://localhost:3000/api/orders
 Content-Type: application/json
 
     {
-      "id": 1,
+      "id": 2,
       "name": "Horst",
       "Surname": "Frank",
```
Comment (fre-ben) [inline server/samples.http:8]: Sounds familiar... 🤔

File: components/table/table.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        </tr>
+        <tr>
+          <td className={styles.table}>{id}</td>
+          <td className={styles.table}>{address}</td>
+          <td className={styles.table}>{sale}</td>
+        </tr>
+        <tr>
+          <td className={styles.table}>{id}</td>
+          <td className={styles.table}>{address}</td>
+          <td className={styles.table}>{sale}</td>
```
Comment (fre-ben) [inline components/table/table.tsx:41]: You could `.map()` these entries later on 💯

**Review by fre-ben — APPROVED** [review]
LGTM 🍕 
Just a few suggestions :)

File: components/table/table.module.css (hunk 1 above)
Comment (mrana17) [reply components/table/table.module.css]: good idea! thanks!

File: components/table/table.tsx (hunk 2 above)
Comment (mrana17) [reply components/table/table.tsx]: absolutely!

**Pull request merged**